src/pages/index.html

```diff
@@ -0,0 +1,504 @@
+{% extends "layout.html" %} {% set title = 'Starct' %} {% block content %}
```

Comment:
@shaban011147 @M-Khairy-B The landing page is not responsive, please review.

Reply:
done